docs/keto/guides/userset-rewrites.mdx

```diff
@@ -1,56 +1,59 @@
 ---
 id: userset-rewrites
-title: Define global permission rules using Ory Permission Language
-sidebar_label: Global rules
+title: Define global permission rules using the Ory Permission Language
+sidebar_label: Using the Ory Permission Language
 ---
 
 import CodeFromRemote from "@theme/CodeFromRemote"
+import BrowserWindow from "@site/src/theme/BrowserWindow"
 const file = (path) => `https://github.com/ory/keto/blob/master/contrib/rewrites-example/${path}`
 
-This guide explains how to configure namespaces and relations using the
-[Ory Permission Language](../reference/ory-permission-language).
+This guide explains how to configure namespaces and relationship rules using the
+[Ory Permission Language](../reference/ory-permission-language) (OPL). You can then run fine-grained checks against Ory Keto,
+which are answered based on a combination of OPL and the concrete high-level relationships stored.
 
 The example describes a file store. Individual files are organized in a folder hierarchy, and can be accessed by individual users
 or groups of users. Using the Ory Permission Language you can specify that if a user has access to a folder, the user also has
 access to all files in that folder.
 
-## Setup and Configuration
+## Ory Network Setup
 
-First, [install Keto](../install.mdx). Next, create a `keto.yaml` file with this basic content:
+The fastest way to get started with Ory Permissions is using [Ory Console](https://console.ory.sh/). You can register for an Ory
+Network account for free.
 
-<CodeFromRemote src={file("keto.yaml")} title="keto.yaml" />
+After logging in, navigate to `Namespaces & Rules` in the sidebar, and to the `Permisson Rules` tab.
 
-In the `namespaces` key, you see the line `config: file://./namespaces.keto.ts`. This needs to point to your namespace
-configuration in the Ory Permission Language.
+<BrowserWindow url="https://console.ory.sh/projects/3afd6339-b999-4649-a2f6-740736775f80/permissions/configuration">
 
-:::info
+![Custom Domain Cors disabled](../_static/opl_editor.png)
 
-Namespace configurations without the Ory Permission Language are still supported by either specifying the namespaces directly or a
-"naked" URI, e.g. `namespaces: file://.namespaces.yaml`.
+</BrowserWindow>
```

Comment:
This should be wrapped in mdx-code-block I think

Reply:
Why? This is not code we will show to the user, but code to generate the browser window. I'm confused.